R/functions.R

```diff
@@ -3,6 +3,44 @@
 #' Return id of the planning googlesheet
 gs_id <- function() "1HLtyGK_csi5W_v7XChxgTuVjS-RKXqc0Jxos1RBqpwk"
 
+
+#' Sanity checks for the google sheet.
+#' This function is used in data pipe-lines to check that everything
+#' in the google sheet is as expected.
+#' It will throw errors or warnings if something goes wrong.
+#'
+#' @details the function can be used in a dplyr pipe, which is why
+#' it requires and returns the `data` argument.
+#' @param data a data frame (or derivative) loaded from a google sheet
```

Comment:
I guess it's OK to not use whitelines between roxygen sections (in this case you could do it above `@param`), but as it enhances code readability and it's often done by others, I recommend to do so. Typically we don't insert whitelines between `@param` lines, since these belong to the same section.